src/chunk.c

```diff
@@ -3240,6 +3245,15 @@ ts_chunk_drop_process_materialization(Oid hypertable_relid,
 /* Continuous agg materialization hypertables can be dropped
  * only if a user explicitly specifies the table name
  */
+Datum
+ts_pg_chunk_drop(PG_FUNCTION_ARGS)
+{
+	int32 chunk_id = PG_GETARG_INT32(0);
+	Chunk *chunk = ts_chunk_get_by_relid(chunk_id, 0, true);
+	ts_chunk_drop(chunk, DROP_RESTRICT, DEBUG1);
+	PG_RETURN_INT32(chunk_id);
```

Comment:
First, this has the wrong return type. Second, I don't think it makes sense to return the OID (which I belive is your intention) of an object that no longer exists. Should probably return the fully qualified name, or a boolean, if anything.

Reply:
Returns a string containing the (fully qualified) chunk name. Void could work as well, but this seems to be the most useful value to return.